clang: error: no such file or directory: 'crs'

I'm guessing that I'm missing some build dependency. I've installed clang 8.0 and I've trying to build with

```
ctaggart@rustasia:~/wasi-sysroot$ make WASM_CC=/usr/bin/clang-8
```

But the build ends with:
```
clang: error: no such file or directory: 'crs'
clang: error: no such file or directory: '/home/ctaggart/wasi-sysroot/sysroot/lib/wasm32-wasi/libc.a'
Makefile:225: recipe for target '/home/ctaggart/wasi-sysroot/sysroot/lib/wasm32-wasi/libc.a' failed
make: *** [/home/ctaggart/wasi-sysroot/sysroot/lib/wasm32-wasi/libc.a] Error 1
```

I installed clang using instructions from http://apt.llvm.org/
```
sudo apt-get install clang-8 lldb-8 lld-8
```
Do I need something else?

--- a/README.md
+++ b/README.md
@@ -29,11 +29,13 @@ The easiest way to get started with this is to use one of the
 ## Building from source
 
 To build a WASI sysroot from source, obtain a WebAssembly-supporting C compiler
-(currently this is only clang, though we'd like to support other compilers as well),
+(currently this is only clang 8+, though we'd like to support other compilers as well),
 and then run:
 
-```
-make WASM_CC=/path/to/wasm/supporting/c/compiler
+```sh
+make WASM_CC=/path/to/clang/with/wasm/support \
+     WASM_AR=/path/to/llvm-ar \
+     WASM_NM=/path/to/llvm-nm
 ```
 
 This makes a directory called "sysroot", by default. See the top of the Makefile
@@ -41,7 +43,7 @@ for customization options.
 
 To use the sysroot, use the `--sysroot=` option:
 
-```
+```sh
 /path/to/wasm/supporting/c/compiler --sysroot=/path/to/the/newly/built/sysroot ...
 ```
 

